ui/StatusQ/src/StatusQ/Popups/Dialog/+qt6/StatusFileDialog.qml

```diff
@@ -0,0 +1,37 @@
+import QtQuick 2.15
+import QtQuick.Dialogs
+
+Item {
+    id: root
+
+    property alias title: dlg.title
+    property alias nameFilters: dlg.nameFilters
+    property alias selectedFile: dlg.selectedFile
+    property alias selectedFiles: dlg.selectedFiles
```

Comment:
For backwards compatibility, you could add `fileUrl(s)` and not worry about changing the existing code :)

Reply:
As in:
```qml
// COMPAT with Qt5 API:
property alias fileUrl: selectedFile
property alias fileUrls: selectedFiles
```

Reply:
I decided to do this way and changing the existing code bc I understand we are going to get rid of the qt5 version at some point so better to have property names the most up-to-date possible so that we don't have confusions.

Reply:
yeah, fair enough, was just a suggestion